### PR TITLE
Add plausible events for favorite features

### PR DESCRIPTION
**Pull request by sjaanus**

### CODE REVIEW

File: frontend/src/component/feature/FeatureToggleList/FeatureToggleListTable.tsx [hunk 1]
```diff
@@ -172,7 +173,7 @@ export const FeatureToggleListTable: VFC = () => {
                 accessor: 'description',
             },
         ],
-        [isFavoritesPinned]
+        [isFavoritesPinned, uiConfig?.flags?.favorites]
```
Comment (sjaanus) [inline frontend/src/component/feature/FeatureToggleList/FeatureToggleListTable.tsx:176]: Fixed a small bug, that favorites were not updating, due to missing dependency.

File: frontend/src/hooks/api/actions/useFavoriteFeaturesApi/useFavoriteFeaturesApi.ts [hunk 2]
```diff
@@ -51,6 +58,11 @@ export const useFavoriteFeaturesApi = () => {
                     title: 'Toggle removed from favorites',
                     type: 'success',
                 });
+                trackEvent('favorite', {
```
Comment (kwasniew) [inline frontend/src/hooks/api/actions/useFavoriteFeaturesApi/useFavoriteFeaturesApi.ts:61]: why do we have this event in the catch block?

**Review by kwasniew — APPROVED** [review]
One weird track event in the error catch block

File: frontend/src/hooks/api/actions/useFavoriteFeaturesApi/useFavoriteFeaturesApi.ts (hunk 2 above)
Comment (sjaanus) [reply frontend/src/hooks/api/actions/useFavoriteFeaturesApi/useFavoriteFeaturesApi.ts:61]: It is in try block. Hmm?

Comment (kwasniew) [reply frontend/src/hooks/api/actions/useFavoriteFeaturesApi/useFavoriteFeaturesApi.ts:61]: sorry weird github formatting

**Pull request merged**